.github/workflows/deploy.yml

```diff
@@ -0,0 +1,89 @@
+name: Deploy to TT02 and Production
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: environment
+        description: Select the environment
+      version:
+        type: string
+        description: Enter the package version
+        required: true
+          
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: altinn/altinn-auth-audit-log:${{ github.event.release.tag_name }}
+  CONTAINER_APP_CONTAINER_NAME: auditlog
+  
+jobs:
+  deploy-tt02:
+    environment: TT02
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+      packages: read
+    steps:
+    - name: Log in to the Container registry
```

Comment:
Why are you logging in to the container registry?

Reply:
it was required when we were using azure container registry (private) at the start and then moved to the github package registry which is now public. I failed to remove it after refactoring

Reply:
removed